index.js

```diff
@@ -0,0 +1,59 @@
+//check for lowercase//
+function checkForLowerCase(password){
+    for (var i = 0; i < password.length; i++){
+        if (password[i] >= "a" && password[i] <="z"){
+            return true
+    }
+    }
+    return false
+}
+//check for password length//
+function checkForLength(password){
+    if (password.length < 8) {
+        return false
+    }
+    return true
+}
+//check for uppercase//
+function checkForUpperCase(password){
+    for (var i = 0; i < password.length; i++){
```

Comment:
multiple for loops :(

try considering using one for loop at the main function level only..